substrate/frame/nomination-pools/src/lib.rs

```diff
@@ -2995,6 +2995,41 @@ pub mod pallet {
 			Ok(Pays::No.into())
 		}
 
+		/// Remove the member if stake falls below the existential deposit.
+		#[pallet::call_index(26)]
+		// FIXME weight functions
+		#[pallet::weight(T::WeightInfo::reap_member_below_ed())]
```

Comment:
you could just put a constant value in here before you create the benchmarks. then the compiler won't complain.